genesis/bootstrappers.go

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2019-2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package genesis
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+
+	"github.com/ava-labs/avalanchego/utils/sampler"
+)
+
+var bootstrappersPerNetworkID map[uint32][]Bootstrapper
+
+func init() {
+	f, err := os.OpenFile("bootstrappers.json", os.O_RDWR, 0600)
+	if err != nil {
+		panic(fmt.Sprintf("failed to read bootstrappers.json %v", err))
+	}
+	err = json.NewDecoder(f).Decode(&bootstrappersPerNetworkID)
+	if err != nil {
+		panic(fmt.Sprintf("failed to decode bootstrappers.json %v", err))
+	}
+}
+
+// Represents the relationship between the nodeID and the nodeIP.
+// Sometimes called "anchor" or "beacon" node.
+type Bootstrapper struct {
+	ID string `json:"id"`
+	IP string `json:"ip"`
```

Comment:
I feel like these should be `ids.NodeID` and `utils.IPDesc` types

Reply:
+1. Now using those types in the struct